mage_ai/api/resources/LogResource.py

```diff
@@ -123,17 +182,6 @@ def get_pipeline_runs():
                     filter(a.id.in_(pipeline_run_ids))
                 )
 
-            if start_timestamp:
```

Comment:
Was that when there were too many logs being loaded so the app crashed? We do still include the time filter automatically, but it's been moved to the `if unix_start_timestamp:` and `if unix_end_timestamp:` blocks. It should still filter out the logs from the timestamp filters so the FE isn't loading and rendering excessive logs.

Reply:
Had just removed this timestamp filter block here since `execution_date` of runs doesn't work well for reliably filtering out certain logs as a user would expect, notably in streaming pipelines but also in standard pipelines as well.

Reply:
Backend will crash when loading too many files into memory. (out of memory error)

Reply:
ah i see